scripts/barclamp_mgmt_lib.rb

```diff
@@ -440,7 +440,7 @@ def bc_install_layout_1_app(from_rpm, bc, bc_path, yaml)
   yml_path = File.join CROWBAR_PATH, 'barclamps'
   yml_created = File.join(yml_path, "#{bc}.yml")
   FileUtils.mkdir yml_path unless File.directory? yml_path
-  FileUtils.cp yml_barclamp, yml_created
+  FileUtils.cp yml_barclamp, yml_created unless yml_barclamp == yml_created
```

Comment:
It happens when this https://github.com/crowbar/crowbar/blob/master/scripts/barclamp_install.rb#L165-L170 is triggert.